src/utils/firebase.js

```diff
@@ -0,0 +1,21 @@
+import firebase from 'firebase/app';
+import 'firebase/database';
+
+//Initialize Firebase
+if (firebase.apps.length === 0) {
+  firebase.initializeApp(firebaseConfig);
+}
+
+const reference = (uuid) => {
+  return firebase.database().ref(`customPlays/${uuid}`);
```

Comment:
Il y a ce warning
> Setting a timer for a long period of time, i.e. multiple minutes, is a performance and correctness issue on Android as it keeps the timer module awake, and timers can only be called when the app is in the foreground. See https://github.com/facebook/react-native/issues/12981 for more info.
(Saw setTimeout with duration 386136ms)

Plus d'explications sur la cause et les conséquences : https://github.com/facebook/react-native/issues/12981#issuecomment-652745831

Une solution possible serait d'utiliser l'api http directement : https://stackoverflow.com/questions/44603362/setting-a-timer-for-a-long-period-of-time-i-e-multiple-minutes/62446792#62446792
(mais c'est du taff)

Il y a ce workaround pour changer l'utilisation de setTimeout : https://github.com/firebase/firebase-js-sdk/issues/97#issuecomment-452568889